### PR TITLE
feat: Round rfqm quote decimal places

### PR DESCRIPTION
**Pull request by rhinodavid**
# Description

Closes MKR-77

Round the price returned by RFQM indicative/firm functions to a configurable (now six) decimal places.

# Testing Instructions

Run RFQM service tests.

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [ ] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [ ] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

File: src/services/rfqm_service.ts [hunk 1]
```diff
@@ -120,6 +120,7 @@ const RFQM_SIGNED_QUOTE_EXPIRY_TOO_SOON = new Counter({
     name: 'rfqm_signed_quote_expiry_too_soon',
     help: 'A signed quote was not queued because it would expire too soon',
 });
+const PRICE_DECIMAL_PLACES = 6;
```
Comment (PirosB3) [inline src/services/rfqm_service.ts:123]: I think there should already be this constant in the codebase?

File: src/services/rfqm_service.ts [hunk 2]
```diff
@@ -204,10 +205,13 @@ export class RfqmService {
...
         const takerAmountInUnit = Web3Wrapper.toUnitAmount(bestQuote.takerAmount, takerTokenDecimals);
         const price = isSelling ? makerAmountInUnit.div(takerAmountInUnit) : takerAmountInUnit.div(makerAmountInUnit);
+        // The way the BigNumber round down behavior (https://mikemcl.github.io/bignumber.js/#dp) works requires us
+        // to add 1 to PRICE_DECIMAL_PLACES in order to actually come out with the decimal places specified.
+        const roundedPrice = price.decimalPlaces(PRICE_DECIMAL_PLACES + 1, BigNumber.ROUND_DOWN);
 
         // Prepare response
         return {
-            price,
+            price: roundedPrice,
```
Comment (PirosB3) [inline src/services/rfqm_service.ts:214]: Should this be a number or a string?

**Review by PirosB3 — APPROVED** [review]

File: src/services/rfqm_service.ts (hunk 2 above)
Comment (rhinodavid) [reply src/services/rfqm_service.ts:214]: The type was `BigNumber` and remains `BigNumber`

File: src/services/rfqm_service.ts (hunk 1 above)
Comment (rhinodavid) [reply src/services/rfqm_service.ts:123]: Looked around asset swapper (https://github.com/0xProject/protocol/search?p=3&q=decimals) and it's all over the place. Gonna press with this.

**Pull request merged**